CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to Spotlight
+
+Spotlight is a collaborative, open source project produced by developers, designers, product owners from several organizations. Spotlight mostly, uses the [contribution model](https://github.com/projectblacklight/blacklight/blob/master/CONTRIBUTING.md) from it's parent project [Blacklight](https://github.com/projectblacklight/blacklight). There are a few differences from the Blacklight process that this document will highlight.
+
+## User experience changes
+
+Spotlight has been designed by a team of user experience designers. While there are a lot of great opportunities to improve the user experience, some contributed changes might change other aspects of user experience that has been accounted. When contributing updates or changes to the Spotlight user experience here are some useful guidelines to follow.
```

Comment:
> While there are a lot of great opportunities to improve the user experience, some contributed changes might change other aspects of user experience that has been accounted.

Suggested rewrite of the second half of this sentence (after the comma):

..., sometimes contributions that seem to make sense in a specific context can negatively affect the broader user experience of Spotlight that we've worked hard over the years to improve.